http_service/models.py

```diff
@@ -60,3 +65,60 @@ def retrieve_model(name):
         LOGGER.info(f"ETAG for {model_url} is ok")
 
     return file_path
+
+
+def classify_bug(
+    model_name, bug_ids, bugzilla_token, expiration=DEFAULT_EXPIRATION_TTL
+):
+    # This should be called in a process worker so it should be safe to set
+    # the token here
+    bug_ids_set = set(map(int, bug_ids))
+    bugzilla.set_token(bugzilla_token)
+    bugs = bugzilla._download(bug_ids)
```

Comment:
Maybe we could use download, so we store the bug data locally and don't have to redownload it if somebody asks for the classification of the same bug again with a different model (since the cache only works for same model / same bug).
We'd need a way to avoid the DB growing forever though.

Reply:
I'm not sure what you mean, do you want to use the bug database from the index to avoid getting data from bugzilla directly? Or introduce a cache layer, like Redis?

Reply:
I mean using `download_bugs` instead of `_download`, which will store the bug IDs locally. But we can consider this as a future enhancement.

Reply:
Once there are in the DB, I don't see an easy way to get individuals bugs from the DB. Moreover it we start storing things on disk, we need to be extra careful about cache invalidation. I would prefer to put it as a future enhancement

Reply:
Let's file an issue for doing that later